frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.scss

```diff
@@ -6,4 +6,8 @@
   &__search {
     margin-left: auto;
   }
+  &__info-icon {
+    margin-left: var(--pf-global--spacer--sm);
+    padding: 0;
```

Comment:
Unnecessary if the button uses prop `isInline`

Reply:
Removed this.